onnxruntime/core/session/allocator_adapters.cc

```diff
@@ -11,6 +12,7 @@ namespace onnxruntime {
 
 namespace {
 constexpr uint32_t kOrtAllocatorReserveMinVersion = 18;
+constexpr uint32_t kOrtAllocatorStatsMinVersion = 23;
```

Comment:
nit: we don't seem to use this anywhere. is it required?

Reply:
if it's not used I wouldn't add it. and if the reserve min version also isn't used I'd remove it as well.

Reply:
Removed.